breakpad-symbols/Cargo.toml

```diff
@@ -18,7 +18,7 @@ minidump-common = { version = "0.9.3", path = "../minidump-common" }
 range-map = "0.1.5"
 nom = "~1.2.2"
 log = "0.4.1"
-reqwest = { version = "0.11.3", features = ["blocking", "gzip"] }
+reqwest = { version = "0.11.6", features = ["blocking", "gzip"] }
```

Comment:
Given your hassle with `openssl` I'm questioning whether having the HTTP dependency down here in `breakpad-symbols` is the best choice or not. It is nice to be able to have a `HTTPSymbolProvider` available, and crates that punt entirely and give you a generic "bring your own HTTP implementation" API are super awkward to work with, but `reqwest` also hides a pretty big dependency tree behind it.

It's probably not worth spending too much time on here. In the long run it would be better to just make `minidump-processor` interop with `symbolic`, since Sentry's set of crates have solved all of these problems.

Reply:
n.b.: you may have seen the discussions about merging this `breakpad-symbols` crate with Sentry's equivalent. I originally thought the best plan would be to throw away my crate and use theirs, but it sounds like they don't love theirs either. Now that you've put additional work into this one and it is at least as useful as the canonical Breakpad implementation, I think there's a pretty strong argument for blessing this one as canonical.

Reply:
Interesting thoughts I dare not act on at this precise moment.